src/resolvers/types/deployment/index.js

```diff
@@ -114,18 +111,40 @@ export async function deployInfo(parent) {
   // Build the registry request URL.
   const uri = `http://${platformReleaseName}-registry.${namespace}:${registryPort}/v2/${repo}/tags/list`;
   log.debug(`Requesting docker tags for ${parent.releaseName} at ${uri}`);
+  const repo = await request({
+    method: "GET",
+    uri,
+    json: true,
+    headers: { Authorization: `Bearer ${dockerJWT}` }
+  });
+  return repo.tags;
+}
 
+/*
+ * Return information about the deployment images.
+ * @param {Object} parent The result of the parent resolver.
+ * @return {Object} The deployment info.
+ */
+export async function deployInfo(parent, args, ctx) {
   try {
-    // Request a list of tags.
-    const repo = await request({
-      method: "GET",
-      uri,
-      json: true,
-      headers: { Authorization: `Bearer ${dockerJWT}` }
-    });
-
-    // Generate the response based on list.
-    const latest = findLatestTag(repo.tags);
+    let { tags } = await ctx.db.query.dockerImages(
+      {
+        where: {
+          AND: [{ id: parent.id }, { tag_starts_with: "cli-" }]
```

Comment:
```suggestion
          AND: [{ deployment { id: parent.id } }, { tag_starts_with: "cli-" }]
```
@andriisoldatenko `parent.id` here would be the `Deployment` ID wouldn't it? Thinking it would look something like this suggestion. Haven't tested that though.

Reply:
Yes, this seems more likely to be correct. Also can we just do this as

```suggestion
          { deployment: { id: parent.id }, tag_starts_with: "cli-" }
```

Reply:
fixed.